src/payment/strategies/ppsdk/ppsdk-strategy.ts

```diff
@@ -43,7 +47,25 @@ export class PPSDKStrategy implements PaymentStrategy {
         return this._store.getState();
     }
 
-    async finalize(_options?: PaymentRequestOptions): Promise<InternalCheckoutSelectors> {
+    async finalize(options?: PaymentRequestOptions): Promise<InternalCheckoutSelectors> {
+        const bigpayBaseUrl = this._store.getState().config.getStoreConfig()?.paymentSettings.bigpayBaseUrl;
```

Comment:
will do 👍

Reply:
I found that we already have `getStoreConfigOrThrow` which guarantees the existence of `bigpayBaseUrl` and throws the same error I would've with a new `getPaymentHostOrThrow` method. So rather than create `getPaymentHostOrThrow` I've reused `getStoreConfigOrThrow`.

Here: https://github.com/bigcommerce/checkout-sdk-js/pull/1188/commits/b3c113660963368e41c7c96525af998997cece4c#diff-5b451c15e917a03094dd5d38df6c638a374c0b98d83522b1934dfac06652f659R24